invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Headline/Headline.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { Button, Container, Divider, Grid, Header } from 'semantic-ui-react';
+import { HomeSearchBar } from '../HomeSearchBar';
+import { FrontSiteRoutes } from '../../../../routes/urls';
+import { Link } from 'react-router-dom';
+
+export default class Headline extends Component {
+  render() {
+    if (this.props.renderElement) {
+      return this.props.renderElement(this.props);
+    }
+    console.log(this.props);
```

Comment:
Console stuff, I think it has grown quite big this render method, can be dried out a bit.